vecgeom.spec

```diff
@@ -24,7 +24,6 @@ cmake ../%{n}-%{realversion} \
   -DCMAKE_BUILD_TYPE=Release \
   -DNO_SPECIALIZATION=ON \
   -DBACKEND=Scalar \
-  -DGEANT4_USE_USOLIDS="CONS;POLYCONE;POLYHEDRA" \
```

Comment:
@iarspider , if I understad the @civanch comment https://github.com/cms-sw/cmsdist/issues/7734 , then we only need to remove the changes of `vecgeom.spec`. So I think this change we still need ... right @civanch ? I might be wrong, so please wait for @civanch response